datavault-broker/src/main/java/org/datavaultplatform/broker/controllers/VaultsController.java

```diff
@@ -295,6 +295,13 @@ public Vault checkVaultRetentionPolicy(@RequestHeader(value = "X-UserID", requir
         return vaultsService.checkRetentionPolicy(vaultID);
     }
 
+    @RequestMapping(value = "/vaults/{vaultid}/actual", method = RequestMethod.GET)
+    public Vault getActualVault(@RequestHeader(value = "X-UserID", required = true) String userID,
```

Comment:
This name confused me at first - because _obviously_ there's no existing method called `getVault` that returns something that isn't a vault. And then I checked the code. I think tidying up the name of the other one is beyond what we should be doing but it might be worth sticking a TODO here to tidy up the names

Reply:
Not sure what else to call it, but it's really returning a "managed" (i.e. in the sense of a Hibernate managed entity) vault. I'll name it `getVaultRecord(...)` to indicate that it's a database record as opposed to `getVault(...)`, which... isn't.